### DISCUSSION TITLE
"execute shell" function in Rancher doesn't work

### DISCUSSION THREAD

**Discussion by xjtupdy** [vectordotdev/vector#19034] [General]
### Problem

I deployed Vector on our cluster using the helm chart according to the guide below.
https://vector.dev/docs/setup/installation/package-managers/helm/

But when I click "execute shell" on Rancher dashboard, it doesn't work(Disconnected). While "view log" is working well.

Are there any values in helm chart template that have influence on it?  

### Version

0.33.0

**Comment by StephenWakely** [comment]
The default image specified in our [helm chart](https://github.com/vectordotdev/helm-charts/blob/fff2470bbc7c299c9c2717b35a642ee9bb1906bf/charts/vector/Chart.yaml#L21) is `0.33.1-distroless-libc` which doesn't have a shell included.

You would need to use a different image that does include a shell - `timberio/vector:0.33.1-debian` or `timberio/vector:0.33.1-alpine`.

**Reply by xjtupdy** [reply]
oh, I will try. 
Thank you.
